WalletWasabi/WebClients/Coinstamp/CoinstampExchangeRateProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using WalletWasabi.Backend.Models;
+using WalletWasabi.Interfaces;
+
+namespace WalletWasabi.WebClients.Coinstamp
+{
+	public partial class CoinstampExchangeRateProvider : IExchangeRateProvider
```

Comment:
But you read from bitstamp.net

Reply:
Ohhh that's what @yahiheb was telling me but I didn't get him.